.github/workflows/ci.yml

```diff
@@ -42,12 +42,11 @@ jobs:
     steps:
       - uses: jakebailey/pyright-action@v1
 
-  test:
```

Comment:
should we move this to the backend part? It's not really testing anything just installing. Then there's the question if we need this stuff at all (as well for the frontend) since our integration tests should fail if the install fails. Can likely save some compute here without loosing test coverage.

Reply:
sgtm, we probably want python unit tests at some point but can add that back later.